Cargo.lock

```diff
@@ -2535,31 +2524,36 @@ checksum = "2a60c7ce501c71e03a9c9c0d35b861413ae925bd979cc7a4e30d060069aaac8d"
 
 [[package]]
 name = "mime_guess"
-version = "2.0.3"
+version = "2.0.4"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "2684d4c2e97d99848d30b324b00c8fcc7e5c897b7cbb5819b09e7c90e8baf212"
+checksum = "4192263c238a5f0d0c6bfd21f336a313a4ce1c450542449ca191bb657b4642ef"
 dependencies = [
  "mime 0.3.16",
  "unicase 2.6.0",
 ]
 
 [[package]]
 name = "minidom"
-version = "0.13.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "332592c2149fc7dd40a64fc9ef6f0d65607284b474cef9817d1fc8c7e7b3608e"
+version = "0.14.0"
+source = "git+https://github.com/lrubiorod/xmpp-rs-fork#c0b6a6eaf7b03fa043c5b03e5aaabef897e61ddf"
```

Comment:
Forked repos should belong to Witnet organization to reduce obscurity.

Reply:
Let's move it to https://github.com/witnet/xmpp-rs-fork